S4TF_Tutorial_2.ipynb

```diff
@@ -210,7 +210,7 @@
         "\n",
         "### Select the model\n",
         "\n",
-        "We need to select the kind of model to train. We will use simplest possible model we can, a Dense network. Since the problem is straightforward, this network will require only a single layer, with a single neuron. \n",
+        "We need to select the kind of model to train. We will use simplest possible model we can, a Dense network. Since the problem is straightforward, this network will require only a single layer, with a single neuron.\n",
         "\n"
```

Comment:
Below this, "Let's define an CelsiusToFahrenheit" should be "Let's define a CelsiusToFahrenheit struct"